x/ibc/03-connection/types/version.go

```diff
@@ -50,9 +50,6 @@ func ValidateVersion(encodedVersion string) error {
 		return sdkerrors.Wrapf(err, "failed to unmarshal version string %s", encodedVersion)
 	}
 
-	if strings.TrimSpace(version.Identifier) == "" {
```

Comment:
I think the issue was only referring to channels? Correct me if I'm wrong @cwgoes, @colin-axner

Reply:
correct we need this check